demos/reduced-motion/app.js

```diff
@@ -1,6 +1,72 @@
+// DOM Elements
+const mapEl = document.querySelector("arcgis-map");
```

Comment:
Unrelated to the code, but can the basemap be changed to a dark-grey or less prominent basemap? It seems like the data gets a bit washed out in the sample with the high contrast basemap.

Reply:
Done in the Map Viewer